examples/esp32c6/README.md

```diff
@@ -0,0 +1,39 @@
+### ESP32-C6 RTIC template
+This crate showcases a simple RTIC application for the ESP32-C6.
+
+## Prerequisites
+
+# Nightly Rust
+The ESP32-C6 HAL requires a nightly build of Rust.
```

Comment:
FYI this is no longer the case, and the HAL can be built using `stable` Rust.

(I imagine this is just an artifact from the previous PR on which this one is based, given its age)